### PR TITLE
added dynamic network and improvements music widgets

### PR DESCRIPTION
**Pull request by Deathemonic**
added a dynamic getnetwork and made some improvements on the dynamic music player plus added back the Now Playing (on Browser) indicator

### CODE REVIEW

**Review by Deathemonic — COMMENTED** [review]
I think this should, Idk about getSongPosition, also improve the music controls from lock screen and added back the (on Browser) indicator, plus added a dynamic getNetwork so it should get any wifi card name not just wlan0

File: bin/sus [hunk 1]
```diff
@@ -22,3 +22,5 @@ cat << EOF
 ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⣿⣆⡀⠀⠀⠀⠀⠀⠀⢀⣠⣴⡾⠃⠀
 ⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠛⠻⢿⣿⣾⣿⡿⠿⠟⠋⠁⠀⠀⠀
 EOF
+
+# amogus
```
Comment (janleigh) [inline bin/sus:26]: sussy

File: cfg/eww/scripts/getSongPosition [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+    DURATION=$(playerctl metadata mpris:length | sed 's/.\{6\}$//')
+    printf "%0d:%02d " $((POSITION%3600/60)) $((POSITION%60))
+    printf "/ %0d:%02d" $((DURATION%3600/60)) $((DURATION%60))
+elif [ $not_support -gt 0 ]; then
+    echo "Player does not support possition"
+else
+    echo "You don't have Spotify Premium."
+fi
+
+# IDK maybe fix this I guess
```
Comment (janleigh) [inline cfg/eww/scripts/getSongPosition:35]: fixed in recent commits lmao should discard this on the pr

**Review by janleigh — CHANGES_REQUESTED** [review]

File: bin/sus (hunk 1 above)
Comment (Deathemonic) [reply bin/sus:26]: that's kinda sus right there

File: cfg/eww/scripts/getSongPosition (hunk 2 above)
Comment (Deathemonic) [reply cfg/eww/scripts/getSongPosition]: yeah my method is kinda hacky

**Pull request merged**